app/models/licence_facade.rb

```diff
@@ -5,7 +5,7 @@ class LicenceFacade
   def self.create_for_licences(licences)
     api_data = get_licence_artefacts(licences)
     licences.map do |l|
-      matching_api_data = api_data['results'].find { |d| l.gds_id.to_s == d['details']['licence_identifier'] }
+      matching_api_data = api_data['results'].find { |d| l.gds_id.to_s == d['licence_identifier'] }
```

Comment:
Yeah, I didn't want to change things too much so it was clearer from the diff what it takes to replace the Content API calls, but I will fix this one as it's probably ok.